Automattic/wp-calypso
# A4A > Migration: Implement Commissions empty view

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1442

## Proposed Changes

This PR:

- Refactors the `Migrations` components by moving them under the `primary` folder to follow the standard we have decided on.
- Refactors the `StepSection` & `StepSectionItem` components by moving them to the `components` folder since we use them in multiple sections.
- Implements the empty state view for the Commissions page in Migrations.

## Why are these changes being made?

* To keep the codebase clean
* To create an empty state for Commissions

## Testing Instructions

* Open the A4A live link
* Go to Referrals(Dashboard, Payment Settings, FAQ), Teams, Partner Directory & Client: Subscriptions, and verify that the refactoring has not caused any issues.
* Append the URL with ?flags=-a4a-tracking-site-migrations > Go to Migrations > Verify the page loads as expected. 
* Remove the feature flag > Refresh the page > Go to Migrations: Overview > Verify that the page loads as expected.
* Click the Commissions sub-menu item and verify that the below is displayed. Note: The buttons won't do anything now.

<img width="1728" alt="Screenshot 2024-11-11 at 11 32 43 AM" src="https://github.com/user-attachments/assets/6d3c17d1-3c4c-4d0f-9b57-1d78db09093e">


## Pre-merge Checklist
